types/react-is/index.d.ts

```diff
@@ -10,17 +11,22 @@ import { ReactElement, ReactType } from 'react';
 
 export function typeOf(value: any): symbol | undefined;
 export function isValidElementType(value: any): value is ReactType;
-
+export function isAsyncMode(value: any): value is ReactType;
```

Comment:
Could you clarify why the return type is not `ReactElement<any>`?
Looking at the docs, function returns `true` if value is a JSX element containing `React.unstable_AsyncMode` type.

Reply:
Apologies and nice catch. Have added a new commit addressing this.